hw9/modules/artist.ts

```diff
@@ -0,0 +1,10 @@
+import type {MainClassesTypes} from "./_typesGuardMainClass";
+
+export class Artist{
+    constructor(
+        public name: MainClassesTypes, 
```

Comment:
name can have `object`, `array`, and `Date` types?

Reply:
fixed